lib/src/utils.dart

```diff
@@ -2,6 +2,7 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 import 'package:analyzer/dart/ast/ast.dart';
+import 'package:analyzer/dart/ast/standard_resolution_map.dart';
```

Comment:
Does this effectively bump the minimum pkg/analyzer version?

Any pubspec changes required?